### PR TITLE
Overhaul

### PR DESCRIPTION
**Pull request by patrickkwang**
Changes:
* Remove a bunch of old, unused stuff
* Refactor things for clarity
* Fix a few bugs
* Speed things up - including dumbing down merging, for now
* Fully parallelize qgraph traversal
  All prioritization and throttling is now done on a per-KP basis by trapi-throttle.

### CODE REVIEW

File: strider/compatibility.py [hunk 1]
```diff
@@ -186,11 +181,11 @@ async def load_curies(self, *curies: list[str]):
         url_base = f"{settings.normalizer_url}/get_normalized_nodes"
         try:
             async with httpx.AsyncClient(timeout=None) as client:
-              response = await client.post(
-                  url_base,
-                  json = {"curies": list(curies)},
-              )
-              response.raise_for_status()
+                response = await client.post(
```
Comment (cbizon) [inline strider/compatibility.py:184]: Are these indents intended?

Comment (patrickkwang) [reply strider/compatibility.py:184]: Yes. This was improperly indented.

**Review by uhbrar — CHANGES_REQUESTED** [review]
Getting an error message when running tests locally:

`ImportError while importing test module '/app/tests/test_query.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_query.py:10: in <module>
    from strider.server import APP, lookup
strider/server.py:23: in <module>
    from .fetcher import Binder
strider/fetcher.py:21: in <module>
    import aiostream
E   ModuleNotFoundError: No module named 'aiostream'
_____________________ ERROR collecting tests/test_server.py ______________________
ImportError while importing test module '/app/tests/test_server.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_server.py:19: in <module>
    from strider.server import APP
strider/server.py:23: in <module>
    from .fetcher import Binder
strider/fetcher.py:21: in <module>
    import aiostream
E   ModuleNotFoundError: No module named 'aiostream'`

I think references to all of the deleted files haven't been fully removed, so we should probably do that before merging.

**Comment by patrickkwang** [comment]
Whoops, I will add aiostream to the requirements. Also, it seems that our automated tests are not failing properly.

**Comment by uhbrar** [comment]
Yeah, I'm not sure why the Github automated tests didn't catch that. Seems like it shouldn't be saying that all checks pass. Also, it seems like Strider takes much longer to build locally now? I don't know whether that is an issue or expected behavior, just something I noticed.

**Comment by uhbrar** [comment]
Well, luckily, Github is reporting the same thing as I am this time. I have one test failing locally still. "test_self_edge" in test_server.py is failing.

**Review by uhbrar — APPROVED** [review]

**Pull request merged**